test/scripts/health_check.sh

```diff
@@ -1,26 +1,41 @@
 #!/bin/bash
```

Comment:
I'm not sure we'll even ship the `test` directory, but I think we want this script as part of the package, thus better path would be `scripts` dir in the root repo ?